mutiny-wasm/src/lib.rs

```diff
@@ -1049,6 +1051,37 @@ impl MutinyWallet {
         Ok(())
     }
 
+    /// Checks whether or not the user is subscribed to Mutiny+.
+    ///
+    /// Returns None if there's no subscription at all.
+    /// Returns Some(u64) for their unix expiration timestamp, which may be in the
+    /// past or in the future, depending on whether or not it is currently active.
+    #[wasm_bindgen]
+    pub async fn check_subscribed(&self) -> Result<Option<u64>, MutinyJsError> {
+        Ok(self.inner.node_manager.check_subscribed().await?)
+    }
+
+    /// Gets the subscription plans for Mutiny+ subscriptions
+    #[wasm_bindgen]
+    pub async fn get_subscription_plans(&self) -> Result<JsValue /* Vec<Plan> */, MutinyJsError> {
+        let plans = self.inner.node_manager.get_subscription_plans().await?;
+
+        Ok(JsValue::from_serde(&plans)?)
+    }
+
+    /// Subscribes to a Mutiny+ plan with a specific plan id.
+    ///
+    /// Returns a lightning invoice so that the plan can be paid for to start it.
+    #[wasm_bindgen]
+    pub async fn subscribe_to_plan(&self, id: u8) -> Result<MutinyInvoice, MutinyJsError> {
+        Ok(self.inner.node_manager.subscribe_to_plan(id).await?.into())
+    }
+
+    /// Pay the subscription invoice. This will post a NWC automatically afterwards.
+    pub async fn pay_subscription_invoice(&self, inv: &Invoice) -> Result<(), MutinyJsError> {
```

Comment:
I think this should take a bolt11 string arg, I don't believe I can construct an `Invoice` type from the js side